tests/e2e/config/bootstrap.js

```diff
@@ -52,6 +53,8 @@ const pageEvents = [];
 
 // The Jest timeout is increased because these tests are a bit slow
 jest.setTimeout( PUPPETEER_TIMEOUT || 100000 );
+// Set default timeout for individual expect-puppeteer assertions. (Default: 500)
+setDefaultOptions( { timeout: 2000 } );
```

Comment:
This _can_ make local development a bit slower to fail though, so it might be better to have this only set on Travis?

If this is set a lot lower do you see the same failures locally as on Travis? I tried this locally but couldn't get it to fail locally with this.

Reply:
Not sure we'll even keep this, so far it doesn't seem to make a difference.